upload-s3/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Upload S3'
+description: 'Uploads package to S3'
+inputs:
+  bucket:
+    required: true
+  product:
+    required: true
+  branch:
+    required: true
+  build:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create and release
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/release.tar.gz --body release.tar.gz
+      shell: bash
+    - name: Upload installer features
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/install/installer-features.txt --body installer-features.txt
+      shell: bash
+    - name: Update latest build
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/latestBuild --body latestBuild
+      shell: bash

```

Comment:
Alt: `aws s3 sync --delete` could help solve the rerun problem.

Reply:
Maybe - I think it would be nicer to fail completely if there are already files, instead of replacing one build with a potentially different build. I'll think about this more and maybe come back in a future PR.